mod.go

```diff
@@ -72,18 +75,32 @@ func init() {
 	}
 
 	Logger = Logger.Level(level)
-
 	PromCollectors = append(PromCollectors, promWarns, promErrs)
+
+	logFile := os.Getenv(EnvLogFile)
+	if len(logFile) > 3 {
```

Comment:
why `> 3` ?

Reply:
why not :-) Just a poor-man's filter to avoid non-sense log file name.  We could put a rich-man's filter with a regex. Putting nothing means that we allow anything like even a dot . or a hidden character. Not sure if we want to create file like that ?